tools/cd_scripts/package_gcsfuse.sh

```diff
@@ -0,0 +1,107 @@
+#! /bin/bash
+# Copyright 2023 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script will create gcsfuse package for both amd64 and arm64 machines.
+# After creation of package we are uploading it on gcs bucket.
+
+# The script is taking around 20-25 minutes on n2-standard-32 machine.
+
+# The script will create following packages and upload it on the bucket.
+# gcsfuse_RELEASE_VERSION_amd64.deb
+# gcsfuse_RELEASE_VERSION_arm64.deb
+# gcsfuse-RELEASE_VERSION-1.aarch64.rpm
+# gcsfuse-RELEASE_VERSION-1.x86_64.rpm
+
+set -e
+# create-gcsfuse-packages VM is a fixed VM in GCP project gcs-fuse-test
+
+# Function to fetch metadata value of the key.
+function fetch_meta_data_value() {
+  metadata_key=$1
+  # Fetch metadata value of the key
+  gcloud compute instances describe create-gcsfuse-packages --zone us-west1-b --flatten="metadata[$metadata_key]" >>  metadata.txt
+  # cat metadata.txt.txt
+  # ---
+  #   value
+  x=$(sed '2!d' metadata.txt)
+  #   value(contains preceding spaces)
+  # Remove spaces
+  # value
+  var_to_set_value=$(echo "$x" | sed 's/[[:space:]]//g')
+  # echo $var_to_set_value
+  # value
+  rm metadata.txt
+}
+
+function exit_in_docket_build_failure() {
+    if [[ $? -ne 0 ]]; then
+      echo "docker build fails."
+      exit 1
+    fi
+}
+# Fetch metadata value of the key "RELEASE_VERSION"
+fetch_meta_data_value "RELEASE_VERSION"
+RELEASE_VERSION=$var_to_set_value
```

Comment:
you can use similar construct to avoid using temp variable (var_to_set_value) https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/perfmetrics/scripts/continuous_test/ml_tests/run_and_manage_test.sh#L194